project_proposal/tex/mainpage.tex

```diff
@@ -79,6 +79,33 @@ \section{Methods}
 
 \section{Initial Requirement Analysis}
 
+\subsection{Problem Statement}
+In the table \ref{table:problem-statement}, the finished problem, the problem statement and related questions are listed.
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
+        \hline
+        \textbf{Problem} & The production it not efficient enough fort the brewery
```

Comment:
This reminds me awfully lot like group 05s work. Please read it again and make sure that we are not copying their work.